src/main/java/org/broadinstitute/dsm/model/at/DefaultValues.java

```diff
@@ -0,0 +1,63 @@
+package org.broadinstitute.dsm.model.at;
+
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.ParticipantData;
+import org.broadinstitute.dsm.db.dao.participant.data.ParticipantDataDao;
+import org.broadinstitute.dsm.model.participant.data.NewParticipantData;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DefaultValues {
+
+    private static final Logger logger = LoggerFactory.getLogger(DefaultValues.class);
+
+    private static final String FIELD_TYPE_ID = "AT_GROUP_GENOME_STUDY";
+    private static final String GENOME_STUDY_CPT_ID = "GENOME_STUDY_CPT_ID";
```

Comment:
because AT only has that field which needs to get added and no other default values for the form, don't need to query for the rest of the FieldSettings